client/web/compose/src/components/PageBlocks/RecordListConfigurator.vue

```diff
@@ -463,9 +483,20 @@
               <b-form-group
                 horizontal
                 breakpoint="md"
-                :label="$t('recordList.record.perPage')"
-                label-class="text-primary"
               >
+                <div
+                  class="d-flex align-items-center text-primary p-0"
+                >
+                  <span>
```

Comment:
You should use the `<template #label>` slot instead, since otherwise span changes the styling of the label
That also allows you to use the `label-class="d-flex align-items-center ... ` directly on the form-group.

You can search for `<template #label` in the repo and you'll see other examples of this

Same for all other occurrences of this